src/modules/editor/sagas.ts

```diff
@@ -205,3 +213,20 @@ function* handleDropItem(action: DropItemAction) {
     yield put(addItem(asset, position))
   }
 }
+
+function* handleScreenshot(_: ScreenshotAction) {
+  try {
+    const screenshot = yield call(() => editorWindow.editor.takeScreenshot())
+    if (screenshot) {
+      const thumbnail = yield call(() => resizeScreenshot(screenshot, 246, 182))
+      if (thumbnail) {
+        const currentProject = yield select(getCurrentProject)
```

Comment:
`currentProject` is `any` here. We may want to add a `currentProject: Project | null`
We need a better way to type `yield select` tho